src/Http/ApiProblem/ApiProblems.php

```diff
@@ -66,6 +66,36 @@ public static function tokenNotSupported(string $detail): ApiProblem
             ->setStatus(400);
     }
 
+    public static function bodyMissing(): ApiProblem
+    {
+        return (new ApiProblem())
+            ->setType('https://api.publiq.be/probs/body/missing')
+            ->setTitle('Body missing')
+            ->setStatus(400);
+    }
+
+    public static function bodyInvalidSyntax(string $detail): ApiProblem
+    {
+        return (new ApiProblem())
+            ->setType('https://api.publiq.be/probs/body/invalid-syntax')
+            ->setTitle('Invalid body syntax')
+            ->setDetail($detail)
+            ->setStatus(400);
+    }
+
+    public static function bodyInvalidData(string $detail, string $jsonPointer): ApiProblem
+    {
+        $problem = (new ApiProblem())
+            ->setType('https://api.publiq.be/probs/body/invalid-data')
+            ->setTitle('Invalid body data')
+            ->setDetail($detail)
+            ->setStatus(400);
+
+        $problem['jsonPointer'] = $jsonPointer;
```

Comment:
Not a big deal but I wonder if we should make our own `ApiProblem` class which has a method to set a `jsonPointer`?

Reply:
I'll check in how many places we already use this `ApiProblem` class and if it's easy to replace them with an instance of our own class. Because creating our own does not seem a lot of work and I see the added value (also in having 1 less dependency, especially for such a simple class), but I'd like to not have 2 classes named `ApiProblem` in our codebase and replacing all occurences might be some work. I'll investigate that first and if it's too much I'll create a follow-up ticket.

Reply:
There are 110 usages, of which 49 in our own code and the rest in the `vendor` directory. So 49 that we would need to replace. Only 21 "new instance creations" though, 4 of which in tests. The rest are just imports, types, and comments. So it seems doable. But maybe something for another PR though, so I'll do that next but not in this PR.

Reply:
Perfect approach. Thanks!